settings.gradle.kts

```diff
@@ -19,6 +19,6 @@ dependencyResolutionManagement {
     }
 }
 
-rootProject.name = "Lasta"
+rootProject.name = "lasta"
```

Comment:
Is this intentional, does this affect anything else?

Reply:
No! It was a mistake, actually struggled with linking the project name to the app in my computer and I had to change it to make it work, now it's fixed !